lib/winston/transports/console.js

```diff
@@ -78,30 +78,29 @@ module.exports = class Console extends TransportStream {
    * Array. For backwards compatibility, stderrLevels defaults to
    * ['error', 'debug'] or ['error'] depending on whether options.debugStdout
    * is true.
-   * @param {mixed} levels - TODO: add param description.
-   * @param {mixed} debugStdout - TODO: add param description.
-   * @returns {mixed} - TODO: add return description.
+   * @param {mixed} levels - An Array of string names of levels whose output
+   *                         should go to stderr
+   * @param {mixed} debugStdout - Whether the 'debug' level should be sent to
+   *                              stdout (default true as of `winston@3`).
+   * @returns {mixed} - An Object containing the levels to send to stderr.
    * @private
    */
   _getStderrLevels(levels, debugStdout) {
```

Comment:
This whole function can be removed since by default *all levels* should now go to stdout unless the user explicitly provides logging levels they would like to go to stderr.

Reply:
Gotcha.  The constructor now takes stderrLevels as an optional string array as one of the potential options.  Defaults to {}.